test/distributed/test_c10d_spawn.py

```diff
@@ -32,14 +32,21 @@
 NO_NCCL = not hasattr(c10d, "ProcessGroupNCCL")
 
 
+def create_device():
```

Comment:
this function seems appear multiple times. It can go to `torch.testing._internal.common_distributed` to dedup.